src/map/unit.c

```diff
@@ -682,97 +675,108 @@ static int unit_walktobl_timer(int tid, int64 tick, int id, intptr_t data)
 	return 0;
 }
 
-// Chases a tbl. If the flag&1, use hard-path seek,
-// if flag&2, start attacking upon arrival within range, otherwise just walk to that character.
-static int unit_walktobl(struct block_list *bl, struct block_list *tbl, int range, int flag)
+/**
+ * Makes the unit of @p bl walk up to the target @p tbl.
+ * See @p flag for additional actions or options.
+ * @param bl block_list of unit to move
+ * @param tbl block_list of target unit to move to
+ * @param range stop walking once within a distance of @p range with @p tbl
+ * @param flag *flag* parameter with following options:
+ *  - `& 1` -> 1/0 = easy / hard
+ *  - `& 2` -> 1 = chase to attack upon arrival within range.
```

Comment:
Please add `@n` to force line breaks in the created documentation.

```suggestion
 * @param flag *flag* parameter with following options: @n
 *  - `& 1` -> 1/0 = easy / hard @n
 *  - `& 2` -> 1 = chase to attack upon arrival within range.
```